gcloud-sync

```diff
@@ -121,7 +126,7 @@ EOT;
         }
         $this->_bucket = $bucket;
 
-        // Sync Magento configuration
+        // Sync Magento configuration, sync composer.* files for magento 2
```

Comment:
We don't need to specify that we're syncing composer.* files, it's part of the configuration switch case for both M1 and M2. Unnecessary comment